shared/actions/chat2/index.js

```diff
@@ -1593,14 +1593,13 @@ function* previewConversationFindExisting(state, action) {
       visibility: RPCTypes.commonTLFVisibility.private,
       ...params,
     })
-    yield Saga.put(
-      Chat2Gen.createMetasReceived({
-        metas: (results.uiConversations || []).map(Constants.inboxUIItemToConversationMeta),
-      })
-    )
-    yield* previewConversationAfterFindExisting(state, action, results, users)
+    const resultMetas = (results.uiConversations || [])
+      .map(row => Constants.inboxUIItemToConversationMeta(row))
+      .filter(Boolean)
+    yield Saga.put(Chat2Gen.createMetasReceived({metas: resultMetas}))
+    yield* previewConversationAfterFindExisting(state, action, resultMetas, users)
   } else {
-    yield* previewConversationAfterFindExisting(state, action, undefined, [])
+    yield* previewConversationAfterFindExisting(state, action, [], [])
```

Comment:
You sure this is ok?

Reply:
The only usages of that `[]` value are in this diff. As `results` in `previewConversationAfterFindExisting`. And I tried a couple things. Should be ok